src/components/enterprise/validate.js

```diff
@@ -0,0 +1,36 @@
+import { isEmail, isMobilePhone, isInt, isByteLength } from "validator";
+
+const validate = (values) => {
+  const requiredMsg = "This field is required";
+  const errors = {};
+
+  if (!values.name) errors.name = requiredMsg;
+  if (!values.abn) errors.abn = requiredMsg;
+  if (!values.emailDomain) errors.emailDomain = requiredMsg;
+  if (!values.contactEmail) errors.contactEmail = requiredMsg;
+  if (!values.contactPhoneNumber) errors.contactPhoneNumber = requiredMsg;
+  if (!values.payrollContact) errors.payrollContact = requiredMsg;
+  if (!values.payrollInformation) errors.payrollInformation = requiredMsg;
+  if (!values.numberOfEmployers) errors.numberOfEmployers = requiredMsg;
+
+  if (!isEmail(values.contactEmail))
+    errors.contactEmail = "Please enter a valid email";
+
+  if (!isMobilePhone(values.contactPhoneNumber, "en-AU"))
```

Comment:
This will fail on a valid landline number (Otherwise, good work on the rest of the validations)

Reply:
If we're just asking for mobile numbers, then it needs to be specified in the form, if not then this should include something for landlines (modify this regex?: `/^(\+?61|0)4\d{8}$/`